runtime.md

```diff
@@ -136,11 +140,13 @@ Attempting to send a signal to a container that is neither [`created` nor `runni
 This operation MUST send the specified signal to the container process.
 
 ### <a name="runtimeDelete" />Delete
-`delete <container-id>`
+`delete <container-id> [<container-id> ...]`
```

Comment:
https://github.com/opencontainers/runtime-spec/blob/5610abdb9fac3b48b2c0ba6216d77320cbbbfb6f/runtime.md?plain=1#L95

I agree with @cyphar 

This seems to be the scope of the runc's implementation details.